gammapy/scripts/tests/test_make_bg_cube_models.py

```diff
@@ -0,0 +1,24 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from astropy.tests.helper import pytest
+from ...datasets import get_path
+from ..make_bg_cube_models import main as make_bg_cube_models_main
+
+
+try:
+    import scipy
+    HAS_SCIPY = True
+except ImportError:
+    HAS_SCIPY = False
+
+
+@pytest.mark.skipif('not HAS_SCIPY')
+@pytest.mark.xfail
+def test_make_bg_cube_models_main(tmpdir):
```

Comment:
Done. I added a dummy dataset generator, otherwise the test would not work. Now the test is working, however, I'm not sure what the best strategy for the asserts is. Suggestions? Maybe we can discuss in the hangout.